cmd/ingester/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 		preallocConfig   client.PreallocConfig
 		clientConfig     client.Config
 		marshalConfig    encoding.MarshalConfig
+		bigchunkConfig   encoding.BigchunkConfig
```

Comment:
It might be more future proof to make this an `encoding.Config`, and roll in the marshalling config (and the setting of DefaultEncoding from the ingester New())

Reply:
Done.